wolfssl/wolfcrypt/logging.h

```diff
@@ -178,7 +178,7 @@ WOLFSSL_API void wolfSSL_SetLoggingPrefix(const char* prefix);
     WOLFSSL_API void WOLFSSL_MSG_EX(const char* fmt, ...);
     #define HAVE_WOLFSSL_MSG_EX
 #else
-    #define WOLFSSL_MSG_EX(...) WC_DO_NOTHING
+    #define WOLFSSL_MSG_EX() WC_DO_NOTHING
```

Comment:
I suspect this is going to cause problems, given that `WOLFSSL_MSG_EX()` always has at least 1 arg, and sometimes has more than 1.

can we gate the `WOLFSSL_MSG_EX()` definition in some way, so that `WOLFSSL_MSG_EX(...)` is kept on targets that support variadic macros?